tests/test_on_data/test_gin_ecephys.py

```diff
@@ -48,9 +49,16 @@
     MCSRawRecordingInterface,
 )
 
+# enable to run locally in interactive mode
+try:
+    from .setup_paths import ECEPHY_DATA_PATH as DATA_PATH
+    from .setup_paths import OUTPUT_PATH
+except:
+    from setup_paths import ECEPHY_DATA_PATH as DATA_PATH
+    from setup_paths import OUTPUT_PATH
```

Comment:
this logic is done to allow to run tests cases in ipython, which is very convenient for debugging (relative imports are not allowed)

Reply:
How are you running your tests? 
I can run specific tests with the currente setup. Example:
```
pytest tests/test_on_data/test_gin_ecephys.py::TestEcephysNwbConversions
```

You need to be on the top level directory for this to work though.

Reply:
I'd like to run them in python/ipython for debugging: e.g. from `tests/test_on_data/` folder:
```
python test_gin_ecephys.py
```

Reply:
`python -m pytest tests/test_on_data/test_gin_ecephys.py` should work to run from python from that folder or any above.
 

How do you debug though? Do you put traces by hand with pdb?

Reply:
Requesting to add the specific error type to the catcher along with a comment at the line of exception explaining why interactivity needs it